python/paddle/fluid/tests/unittests/test_reduce_op.py

```diff
@@ -243,6 +244,10 @@ def setUp(self):
     def test_check_output(self):
         self.check_output(check_eager=True)
 
+    def test_check_grad(self):
+        # only composite op support gradient check of reduce_max
+        self.check_grad(['X'], 'Out', check_eager=True, only_check_prim=True)
+
     def test_raise_error(self):
```

Comment:
if original test pass fp16, please describe it in pr describes

Reply:
> if original test pass fp16, please describe it in pr describes

done